test/parallel/test-file-write-stream2.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var path = require('path');
-var fs = require('fs');
+const path = require('path');
+const fs = require('fs');
 
 
 var filepath = path.join(common.tmpDir, 'write.txt');
```

Comment:
Nit: While we're in here refactoring anyway, maybe line 79 can be converted from `assert.ok(err.message.indexOf('write after end') >= 0);` to something more like `assert.ok(err.message.includes('write after end'))`?